frontend/src/components/transactions/TransactionOutcome.tsx

```diff
@@ -0,0 +1,76 @@
+import BN from "bn.js";
+import React from "react";
+import { Row, Col } from "react-bootstrap";
+
+import * as T from "../../libraries/explorer-wamp/transactions";
+import Gas from "../utils/Gas";
+import Balance from "../utils/Balance";
+
+export interface Props {
+  transaction: T.TransactionOutcome;
+}
+
+class TransactionOutcome extends React.Component<Props> {
+  render() {
+    const { transaction } = this.props;
+    const gasBurnt = transaction?.outcome
+      ? new BN(transaction.outcome.gas_burnt)
+      : new BN(0);
```

Comment:
Would this work? (also, it seems that `transaction` cannot be undefined or null, so `?` after it is not necessary.

```suggestion
    const gasBurnt = new BN(transaction.outcome?.gas_burnt ?? 0);
```

If it does, update the `tokensBurnt` and other places (if any) as well.